test.js

```diff
@@ -115,6 +119,19 @@ test('POST JSON', async t => {
 	await server.close();
 });
 
+test('baseUrl option', async t => {
+	const server = await createTestServer();
+	server.get('/api/unicorn', (request, response) => {
+		response.end('rainbow');
+	});
+
+	t.is(await ky('unicorn', {
+		baseUrl: server.url + '/api/'
+	}).text(), 'rainbow');
```

Comment:
Can you add a test case for using an absolute URL?

Reply:
You can borrow all of these tests: https://github.com/sindresorhus/got/pull/579/files#diff-bbc23533bfd4986af19324ccf2e7c4baR201 Just to ensure we cover it all.